ui-v2/tests/steps.js

```diff
@@ -83,6 +85,15 @@ export default function(assert) {
       .when('I click "$selector"', function(selector) {
         return click(selector);
       })
+      // TODO: Probably nicer to thing of better vocab than having the 'without " rule'
```

Comment:
Is "thing" here supposed to be "think"?

Reply:
Yup 👍 I'll get that in in the commit with the other one above